Dockerfile

```
@@ -23,13 +23,15 @@ WORKDIR /opt/lib/perl/VEP/Plugins
 
 RUN wget https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/93/Downstream.pm \
```

Should this one also get updated to 95?
```suggestion
RUN wget https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/95/Downstream.pm \
```